src/main/java/com/cloudbees/jenkins/support/SupportAction.java

```diff
@@ -114,6 +130,72 @@ public Localizable getActionBlurb() {
         return SupportPlugin.getComponents();
     }
 
+    public class Bundle {
```

Comment:
should be static or in a separate file - does not use any of the outer classes internals.